test/ctia/flows/hooks/kafka_event_hook_test.clj

```diff
@@ -0,0 +1,167 @@
+(ns ctia.flows.hooks.kafka-event-hook-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.string :as str]
+            [clojure.test :refer [deftest is join-fixtures testing use-fixtures]]
+            [ctia.domain.entities :refer [schema-version]]
+            [ctia.lib.kafka :as lk]
+            [ctia.properties :refer [properties]]
+            [ctim.domain.id :as id]
+            [ctim.schemas.common :as c]
+            [cheshire.core :refer [parse-string]]
+            [ctia.test-helpers
+             [core :as test-helpers :refer [post]]
+             [es :as es-helpers]])
+  (:import [java.util.concurrent CountDownLatch TimeUnit]))
+
+(use-fixtures :once mth/fixture-schema-validation)
+
+(use-fixtures :each
+  (join-fixtures [test-helpers/fixture-properties:clean
+                  es-helpers/fixture-properties:es-store
+                  test-helpers/fixture-properties:kafka-hook
+                  test-helpers/fixture-properties:events-enabled
+                  test-helpers/fixture-ctia
+                  test-helpers/fixture-allow-all-auth]))
+
+(deftest ^:integration test-events-topic
+  (testing "Events are published to kafka topic"
+    (let [results (atom [])
+          finish-signal (CountDownLatch. 3)
+          {:keys [channel-name] :as kafka-config}
+          (get-in @properties [:ctia :hook :kafka])
+          consumer-map
+          (lk/subscribe
+           (fn test-events-kafka-topic-fn [ev]
+             (let [v (:value ev)]
+               (swap! results conj (parse-string v true))
+               (.countDown finish-signal)))
+           30000)
+          ;; await rebalance
+          _ (Thread/sleep 5000)]
```

Comment:
We've been bitten multiple time by those waiting systems inside Travis CI, I would suggest to try to find a better way if possible.